assets/css/00-settings/_typography.scss

```diff
@@ -1,21 +1,21 @@
 // Typography
-$font-regular: Arial, sans-serif;
-$font-heading: Verdana, $font-regular;
-
-$base-font-family: $font-regular;
-$heading-font-family: $font-heading;
+$font-regular: Arial, sans-serif;
+$font-heading: Verdana, $font-regular;
+
+$font-family-base: $font-regular;
+$font-family-heading: $font-heading;
 
 // Font Sizes
-$base-font-size: 1rem;
-$larger-font-size: 1.5rem;
-$large-font-size: 1.25rem;
-$small-font-size: .875rem;
-$smaller-font-size: .8125rem;
-$button-font-size: 1.0625rem;
+$font-size-base: 1rem;
+$font-size-larger: 1.5rem;
+$font-size-large: 1.25rem;
+$font-size-small: .875rem;
+$font-size-smaller: .8125rem;
+$font-size-button: 1.0625rem;
 
 // Line height
-$base-line-height: 1.5;
-$heading-line-height: 1.2;
+$line-height-base: 1.5;
+$line-height-heading: 1.2;
 
 // Line height scale
-$heading-scale: 1.21;
+$line-height-heading-scale: 1.21;
```

Comment:
$heading-scale is geen line-height variabele, maar een ratio waarmee de base font-size wordt vermenigvuldigd in incrementele stappen van h6 naar h1. Aanpassing van deze variabele naam zou ik hier graag terug willen draaien.